helm/flux-app/templates/base/deployment-helm-controller.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: helm-controller
+{{ include "labels.common" . | indent 4 }}
+  name: helm-controller
```

Comment:
I would expect that too, but better to be sure. Regardless the outcome, I think it is worth adding the `{{ .Release.Namespace }}` explicitly, in order to make the templates consistent, because now some have it and others don't, WDYT? Also, without it a dry-run does not show the namespace, at least for me, maybe it depends on the Helm version, which may be sometimes confusing to users who want to see the preview of resources to be created.

Reply:
I'll add, alright